provision/README.md

```diff
@@ -0,0 +1,5 @@
+# Provision
+
+[![GoDoc](https://godoc.org/github.com/golang/gddo?status.svg)](https://godoc.org/github.com/ubclaunchpad/inertia/provision)
+
+This package contains API calls to the [Amazon Elastic Compute Cloud](https://aws.amazon.com/ec2/) web service.
```

Comment:
This package will handle API calls to a variety providers (just haven't gotten around to it yet 😋 ), so it might be better to make this doc a big more generalized. Again see the package's [`doc.go`](https://github.com/ubclaunchpad/inertia/blob/master/provision/doc.go) for the official summary.